.circleci/config.yml

```diff
@@ -88,6 +88,22 @@ commands:
           paths:
             - ~/.cache/yarn
 
+  restore_nx_cache:
+    description: Restore nx cache from previous build
+    steps:
+      - restore_cache:
+          keys:
+            - nx-v4-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - nx-v4-master-{{ checksum "yarn.lock" }}
```

Comment:
Why have different cache keys for different branches? Can we just use the same cache based on `yarn.lock`?

Reply:
@nolanlawson thanks for pointing this out!   

My original thought was that if each branch had its own copy of the cache it would potentially speed up rebuilds per branch.

I realize now that the way it's set up here, it'll only cache when the branch is first created or if the `yarn.lock` file changes.